docs/pages/base/api/snackbar-unstyled.js

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import ApiPage from 'docs/src/modules/components/ApiPage';
+import mapApiPageTranslations from 'docs/src/modules/utils/mapApiPageTranslations';
+import jsonPageContent from './snackbar-unstyled.json';
+
+export default function Page(props) {
+  const { descriptions, pageContent } = props;
+  return <ApiPage descriptions={descriptions} pageContent={pageContent} />;
+}
+
+export function getStaticProps() {
```

Comment:
Please update to latest master, the change for using `getStaticProps` was reverted. We should use `Page.getInitialProps`.

Reply:
Done. 👍